eos-tech-support/eos-user-stress-test

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DATA_FILES_URL=https://github.com/endlessm/eos-qa-test-files/archive/master.zip
```

Comment:
This file does not exist in the repo.  Personally, I would avoid using a zip file, as that is just more work for the developer to maintain the files, and easily forgotten to update.  I don't see an easy way to download a directory of files from GitHub, but since the script knows exactly which files it needs, you could just download the four files separately.

Reply:
Ah, as Travis pointed out, the web server creates this zip file.  As long as we are not putting other stuff into this repo, this makes sense for now.  I'll merge as is.